### ISSUE TITLE
Syntactic Sugar for Definition of Multidimensional Array

### ISSUE THREAD

**Issue by wwlittewayne** [vlang/v#18024]
### Describe the feature

As an example in the doc, we need to define a 3d array like:
    `mut a := [][][]int{len: 2, init: [][]int{len: 3, init: []int{len: 2}}}`
How about to support another style like:
    `mut a := [][][]int{len: [2,3,2]}`
It is much easier to read and write.

### Use Case

To define of a Multidimensional Array is not simple.

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### Version used

0.3.3

### Environment details (OS name and version, etc.)

deepin 20.9

**Comment by ArtemkaKun** [comment]
@JalonSolov does this related to your proposition for arrays syntax change?

**Comment by JalonSolov** [comment]
Not really.  My proposal is more to have a simple common syntax between dynamic and fixed array.
